from sqlite3 import connect, IntegrityError
import os
from datetime import datetime

dir_path = os.path.dirname(os.path.realpath(__file__))
